packages/backend-modules/lead-tracking/README.md

```diff
@@ -0,0 +1,16 @@
+# How to use the import script
+
+To import emails from mail chimp csv use the following command
```

Comment:
```suggestion
To import emails from an exported mail chimp csv-file use the following command
```
Very nitpicky, my first thought upon reading was it would directly import from mailchimp so maybe something like this would be more explicit 🙈